### PR TITLE
adjustments to CMake 

### PR DESCRIPTION
**Pull request by apBUSampK**
[example repository](https://github.com/apBUSampK/COLA-min-example) has been updated too

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -10,31 +10,28 @@ target_include_directories(COLAlib INTERFACE
...
-if(NOT INSTALL_DIR)
-    set(INSTALL_DIR ${PROJECT_SOURCE_DIR}/install)
-endif()
-
 install(TARGETS COLAlib
-        EXPORT COLA
-        PUBLIC_HEADER DESTINATION ${INSTALL_DIR}/include)
+        EXPORT COLAlibExport
+        PUBLIC_HEADER DESTINATION include
+        INCLUDES DESTINATION include) #Needed for proper EXPORT generation
```
Comment (alexsvetlichnyy) [inline CMakeLists.txt:16]: why only one bracket?

**Review by alexsvetlichnyy — COMMENTED** [review]
What is the main difference?

File: CMakeLists.txt (hunk 1 above)
Comment (apBUSampK) [reply CMakeLists.txt:16]: There are two...

**Comment by apBUSampK** [comment]
> What is the main difference?

Correct Export file generation + some more logical names for config/export cmake files.

Now usage is find_package(COLAlib) and target_link_libraries(target PUBLIC COLAlib) for modules, as it is supposed to work. 

And in projects with actual modules you don't even need to link COLA, provided you've written the module correctly.

File: CMakeLists.txt (hunk 1 above)
Comment (alexsvetlichnyy) [reply CMakeLists.txt:16]: okay

**Pull request merged**